Cmake failed on ubuntu xenial

**Describe Your Environment**
- Zoneminder from commit 11d6de1acaaf7a7b707f932e7da2ac2ed46b010c
- Build zoneminder from https://github.com/ZoneMinder/zmdockerfiles/blob/master/development/ubuntu/xenial/Dockerfile


**Describe the bug**
step `cmake` failed

**Debug Logs**
```

Building a static library
Building unit tests: No (default)
-- Optional libraries found: zlib cURL OpenSSL PCRE GCrypt AVFormat AVCodec AVDevice AVUtil SWScale SWResample libVLC
-- Optional libraries not found: LIBJWT libVNC libLive555
-- Running ZoneMinder configuration generator
Generating '/ZoneMinder/src/zm_config_defines.h'
Updating '/ZoneMinder/db/zm_create.sql'
-- ZoneMinder configuration generator completed successfully
-- Configuring done
CMake Error in src/CMakeLists.txt:
  Specified unknown feature "cxx_std_11" for target "zmc".


CMake Error in src/CMakeLists.txt:
  Specified unknown feature "cxx_std_11" for target "zms".


CMake Error in src/CMakeLists.txt:
  Specified unknown feature "cxx_std_11" for target "zm".


CMake Error in src/CMakeLists.txt:
  Specified unknown feature "cxx_std_11" for target "zmu".


-- Generating done
-- Build files have been written to: /ZoneMinder
Service 'zm' failed to build: The command '/bin/sh -c cmake .' returned a non-zero code: 1

```


**INFO**
Failed after https://github.com/ZoneMinder/zoneminder/commit/36253048f598d8665cfc21c7c7ac8ea9fdb24cbf

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -2,7 +2,7 @@
 # Created by mastertheknife (Kfir Itzhak)
 # For more information and installation, see the INSTALL file
 #
-cmake_minimum_required(VERSION 2.8.7)
+cmake_minimum_required(VERSION 3.5.0)
 project(zoneminder)
 file(STRINGS "version" zoneminder_VERSION)
 # make API version a minor of ZM version

--- a/cmake/Modules/ConfigureBaseTargets.cmake
+++ b/cmake/Modules/ConfigureBaseTargets.cmake
@@ -1,13 +1,20 @@
 add_library(zm-compile-option-interface INTERFACE)
 
 # Use -std=c++11 instead of -std=gnu++11
-set(CXX_EXTENSIONS OFF)
+set(CMAKE_CXX_EXTENSIONS OFF)
 
 add_library(zm-feature-interface INTERFACE)
 
-target_compile_features(zm-feature-interface
-  INTERFACE
-    cxx_std_11)
+# The cxx_std_* feature flags were only introduced in CMake 3.8
+# Use to old way to specify the required standard level for older CMake versions.
+# Remove this once we raise the required CMake version.
+if(${CMAKE_VERSION} VERSION_LESS 3.8.0)
+  set(CMAKE_CXX_STANDARD 11)
+else()
+  target_compile_features(zm-feature-interface
+    INTERFACE
+      cxx_std_11)
+endif()
 
 # Interface to set warning levels on targets
 # It gets populated in the compiler specific script

